server/serverMethods.js

```diff
@@ -478,7 +478,7 @@ function ServerMethods(aLogLevel, aModules) {
     }
 
     logger.log('_getUpdatedArchiveInfo: ', aSessionInfo);
-    var minLastUsage = Date.now() - aTbConfig.maxSessionAgeMs;
+    var minLastUsage = Date.now() - aTbConfig.maxSessionAge;
```

Comment:
Date.now() returns a value in milliseconds, so this won't produce the desired result.